tofhir-engine/src/main/scala/io/tofhir/engine/mapping/FhirMappingJobManager.scala

```diff
@@ -84,17 +81,10 @@ class FhirMappingJobManager(
               val jobResult = FhirMappingJobResult(mappingJobExecution, Some(task.mappingRef))
               logger.error(jobResult.toLogstashMarker, jobResult.toString, se)
           }
-        // Halt the execution if error handling type is HALT
-        haltOrContinueExecution(mappingJobExecution, task, se)
-        // Pass the stop exception to the upstream Futures in the chain laid out by foldLeft above
-        case t: FhirMappingJobStoppedException =>
```

Comment:
Since it is a foldLeft chain, When we throw it, it interrupts the next mappingTask, and the function goes to recover part again and again. When we remove this throw t, it is caught in case e: Throwable, and It does not interrupt other mappingTasks.

Reply:
OK, if @YemreGurses agrees that it will not break anything, we can resolve this thread.